circle.py

```diff
@@ -32,7 +33,26 @@ def generate_2D_coordinate(x_min, x_max, y_min, y_max):
     y = y_min + random.random() * (y_max - y_min)
     return (x, y)
 
+'''
+ @brief	    Function that calcultes pi
```

Comment:
So the problem here is actually the tab I beleive. Remove the tab and replace with spaces. That should fix the spacing issue. You can notice these issues a bit better on github (it might not show up on gedit, depending on your settings).